gitenberg/config.py

```diff
@@ -78,6 +78,9 @@ def parse(self):
         global data
         data = yaml.load(self.read())
         data = {} if data is None else data
+        for key, value in os.environ.items():
```

Comment:
I wondered why environment variables are usually all caps: http://pubs.opengroup.org/onlinepubs/000095399/basedefs/xbd_chap08.html

Reply:
Interesting, although Linux definitely supports lowercase ones. Looks like that's mainly a problem on Windows then, and I can't test anything there anyways. I guess I could use key.lower() instead so then it will work in theory? Does any of this get tested on non-Linux platforms anyways? What do you think makes sense?

Specifically, it appears that Windows is case-preserving for environment variables (like paths), so it preserves case but ignores it when comparing. Python deals with this by just forcibly uppercasing all environment variable names on Windows.

Reply:
I'll leave it to you to decide what, if anything, to do.

Reply:
I decided to make it case-insensitive, because that's easy and should work on all OS.